saleor/settings.py

```diff
@@ -276,7 +278,7 @@ def get_bool_from_env(name, default_value):
         'RESULTS_CACHE_SIZE': 100}
 
 ENABLE_SILK = get_bool_from_env('ENABLE_SILK', False)
-if ENABLE_SILK:
+if DEBUG and ENABLE_SILK:
```

Comment:
Sometimes I use silk or debugtoolbar to debug bugs, but these bugs may not be reproduces in ```DEBUG``` mode, should we only enable them in DEBUG mode?

Reply:
Yea I was wondering if this next level of protection is required. Probably not.

Reply:
On the second thoughts, right now debug toolbar is added only in DEBUG mode so no changes to that.
I don't know probably this needs discussion :)